README.md

```diff
@@ -26,7 +26,7 @@ Forklift provides tools to create Foreman/Katello environments for development,
 ### Requirements
 
 * Vagrant - 2.2+ - Both the VirtualBox and Libvirt providers are tested
-* Ansible - 2.9+
+* Ansible - 2.12+
```

Comment:
In https://github.com/theforeman/forklift/issues/1641 it's suggested recent 2.9 should still work.

Reply:
Thanks, I missed the GH issue and got here via https://community.theforeman.org/t/problems-using-forklift/32003

Even if later 2.9+ still works, perhaps it is not ideal to advertise or recommend this fact considering there are multiple EOL versions between 2.9+ and the current maintenance branch.

Using Ansible with Forklift feels pretty different from most other use cases, and we have our own ansible.cfg in version control while Ansible still doesn't have support for merging multiple ansible.cfg files (see https://github.com/ansible/ansible/issues/79506 which looks written by a familiar face fwiw)

Our ansible.cfg also makes `./playbooks/galaxy_collections` the first option for `collections_path` so we're already avoiding installing them system wide when the user follows the current instructions in the README.

So, IMO, we could just add instructions to the README.md which say that, for 99% of users who are running forklift on their workstation and not part of a CI pipeline, it's very easy and reliable to make and activate a python virtual environment, `pip install ansible-galaxy`, `ansible-galaxy collection install -r requirements.yml`, and they are all set.

This entirely avoids a frequent pain point I've seen when trying to help people get up and running with Forklift.